ckan/migration/versions/105_21d03ed1e1ca_add_permission_labels_in_activity_table.py

```diff
@@ -0,0 +1,27 @@
+# encoding: utf-8
+
+"""Add permission labels in activity table
+
+Revision ID: 21d03ed1e1ca
+Revises: 9f33a0280c51
+Create Date: 2023-04-26 14:06:05.086247
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '21d03ed1e1ca'
+down_revision = '9f33a0280c51'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('activity', sa.Column(
+        'permission_labels', sa.ARRAY(sa.Text)))
+
+
+def downgrade():
+    op.drop_column('activity', 'permission_labels')
```

Comment:
@smotornyuk where should activity model migrations live now that the model is in the activity extension?

Reply:
Let's keep activities separately, because it doesn't add any complexity here.

 @vgeorgieva-cstbg, when creating migrations, use `ckan generate migration -p activity -m "Add permission labels in activity table"` command. It will create `ckanext/activity/migration` folder and put all the activity-related migrations there.

In order to apply such migrations you can use either `ckan db upgrade -p activity` to apply migrations from the specific plugin, or `ckan db pending-migrations --apply` if you want to apply all the migrations(that are not applied yet) from all the enabled plugins.

PS you can revert migration as well. use `ckan db downgrade -p activity` if you want to cancel all the activities from the given plugin. Or `ckan db downgrade -p activity -v -1`  if you want to revert only the latest activity(use `-v -2` for two latest activities, etc)

Reply:
@wardi @smotornyuk I moved the migration, but now it doesn't get executed automatically when the tests are ran. What's the best way to fix this?

Reply:
@wardi @smotornyuk I moved the migration, but now it doesn't get executed automatically when the tests are ran. What's the best way to fix this?

Reply:
@vgeorgieva-cstbg Can you have the migration run as part of the extension's test setup? I don't know if there's a module level test setup or if you would have to group all tests into a class and use a class setup method.

Reply:
@wardi Fixed the tests. Anything else left to do?